magnolia/shared/src/main/scala-2.12-2.13/zio/config/magnolia/Descriptor.scala

```diff
@@ -402,11 +402,14 @@ object Descriptor {
             acc orElse f(n)
           }
         case head :: tail =>
+          def nest(name: String)(unwrapped: ConfigDescriptor[Any]) =
```

Comment:
Do we have a test that confirms if the behaviour works? :)

Reply:
added a test, works for you?